dotnet/frontend/Controllers/GreetingController.cs

```diff
@@ -10,24 +12,28 @@ namespace frontend.Controllers
     public class GreetingController : ControllerBase
     {
         private readonly IHttpClientFactory _clientFactory;
+        private readonly Tracer _tracer;
 
-        public GreetingController(IHttpClientFactory clientFactory)
+        public GreetingController(IHttpClientFactory clientFactory, Tracer tracer)
         {
             _clientFactory = clientFactory;
+            _tracer = tracer;
         }
 
         [HttpGet]
         public async Task<string> GetAsync()
         {
-            var current = Activity.Current;
-            current?.AddTag("apple", 1);
-            current?.AddBaggage("avocado", "12");
+            using (var span = _tracer.StartActiveSpan("Preparing Greeting"))

```

Comment:
This is purely cosmetic, but switching this to just a `using var span ...` statement (no braces, parens) is functionally equivalent to the `using` with braces, and a bit less code/noise. In general, since this is a sample, if we use more modern C# features then it might look a little better.

Reply:
Thanks for the note @cartermp ! I updated the syntax in both GreetingController.cs and MessageController.cs.